ThreeBotPackages/zerobot/chat/templates/chat/login.html

```diff
@@ -7,5 +7,6 @@ <h2>Please select one of the following providers to login.</h2>
     {% for provider in providers%}
     <a href="/chat/login?provider={{provider}}"><i class="fa fa-{{provider}}" style="font-size: 33px"></i></a>
     {% endfor %}
+    <a href="/chat/login?provider=3bot">3botlogin</i></a>
```

Comment:
how come this is not part of the providers list ?
Is it a special case ?

Reply:
yes, those are the oauth providers that are handled by oauth proxy. Since 3bot login doesn't follow the same flow and protocol, I can't just add it as a provider. hence I created the threebot login flow I showed you in the other PR.